### PR TITLE
Add the Bullseye layout

### PR DESCRIPTION
**Pull request by cliffpyles**
A bullseye layout is when there is a single child element, and that child is centered both vertically and horizontally in the parent.

resolves: #73

### CODE REVIEW

File: src/patternfly/layouts/Bullseye/docs.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## Overview
+
+Only if needed write a short description with implementation notes. Design and interaction notes are already written on the design specs, don't repeat information.
```
Comment (andresgalante) [inline src/patternfly/layouts/Bullseye/docs.md]: please remove this or describe the component

File: src/patternfly/layouts/Bullseye/docs.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+For example for buttons: Always add a modifier class to add color to the button. Never use the class `.btn` on its own.
+
+## Accessibility
```
Comment (andresgalante) [inline src/patternfly/layouts/Bullseye/docs.md]: if there are no accessibility concerns remove this too

File: src/workspace.scss [hunk 3]
```diff
@@ -154,6 +156,9 @@
   .pf-l-stack__secondary::before {
```
Comment (andresgalante) [inline src/workspace.scss]: sorry I missed this on the other PR. I think this is an unnecessary overhead. We should not do it here.

Comment (cliffpyles) [reply src/workspace.scss]: If we remove the dummy content and we don't include any text in the property, we won't have anyway to annotate what they are looking at.

Comment (cliffpyles) [reply src/workspace.scss]: For clarity I think we should do one or the other.

Comment (cliffpyles) [reply src/workspace.scss]: I'll remove this and add the text back to the components.

Comment (cliffpyles) [reply src/workspace.scss]: To keep this PR in scope, I'll make another PR though.

Comment (cliffpyles) [reply src/workspace.scss]: https://github.com/patternfly/patternfly-next/pull/82

Comment (cliffpyles) [reply src/workspace.scss]: https://github.com/patternfly/patternfly-next/pull/82

**Pull request merged**